clang/test/dpct/asm/prefetch.cu

```diff
@@ -0,0 +1,91 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0, cuda-10.1, cuda-10.2, cuda-11.0, cuda-11.1, cuda-11.2, cuda-11.3, cuda-11.4, cuda-11.5, cuda-11.6, cuda-11.7
```

Comment:
why so many version is disable on test?

Reply:
[Some](https://docs.nvidia.com/cuda/parallel-thread-execution/#instruction-set:~:text=Support%20for%20the%20.tensormap%20qualifier%20is%20introduced%20in%20PTX%20ISA%20version%208.0.) of its features have been recently enabled on PTX ISA 8.0 version (from CUDA 12.0 onwards). Hence, I disabled all the previous versions

Reply:
Created 2 new cases:

1. prefetch_eviction_priority.cu (PTX ISA 7.4)
2. prefetch_tensormap.cu (PTX ISA 8.0)